### PR TITLE
feat: bls key generation

### PR DESCRIPTION
**Pull request by alexbarnsley**
## Summary

https://app.clickup.com/t/86dube053

requires https://github.com/ArkEcosystem/bls-signatures/pull/1 so i can update the submodule to point to develop

## Checklist

- [ ] Documentation _(if necessary)_
- [ ] Tests _(if necessary)_
- [ ] Ready to be merged

### CODE REVIEW

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -14,19 +14,27 @@ jobs:
     strategy:
       max-parallel: 4
       matrix:
+        # python-version: ["3.10"]
```
Comment (ItsANameToo) [inline .github/workflows/test.yml]: remove?

**Comment by alexbarnsley** [comment]
Drafting as we need https://github.com/ArkEcosystem/bls-signatures/pull/1 merged first

**Review by ItsANameToo — APPROVED** [review]

File: .github/workflows/test.yml (hunk 1 above)
Comment (ItsANameToo) [inline .github/workflows/test.yml]:
```suggestion
```

**Comment by ItsANameToo** [comment]
can be updated @alexbarnsley

**Pull request merged**